polaris-ui/src/app/features/cases/presentation/case-details/pdf-tabs/PdfTab.tsx

```diff
@@ -24,6 +24,11 @@ type PdfTabProps = {
   handleRemoveAllRedactions: CaseDetailsState["handleRemoveAllRedactions"];
   handleSavedRedactions: CaseDetailsState["handleSavedRedactions"];
   handleOpenPdfInNewTab: CaseDetailsState["handleOpenPdfInNewTab"];
+  feedbackData: {
```

Comment:
maybe call this `contextData` to make this prop sound less tightly-coupled to the feedback feature